src/cf.data.pre

```diff
@@ -10433,6 +10433,98 @@ DOC_START
 	that the request body is needed. Delaying is the default behavior.
 DOC_END
 
+NAME: http_upgrade_request_protocols
+TYPE: http_upgrade_request_protocols
+LOC: Config.http_upgrade_request_protocols
+DEFAULT: none
+DEFAULT_DOC: Upgrade header dropped, effectively blocking an upgrade attempt.
+DOC_START
+	Controls client-initiated and server-confirmed switching from HTTP to
+	another protocol (or to several protocols) using HTTP Upgrade mechanism
+	defined in RFC 7230 Section 6.7. Squid itself does not understand the
+	protocols being upgraded to and participates in the upgraded communication
+	only as a dumb TCP proxy. Admins should not allow upgrading to protocols
+	that require a more meaningful proxy participation.
+
+	Usage: http_upgrade_request_protocols <protocol> allow|deny [!]acl ...
+		protocol = OTHER | name[/version]
```

Comment:
This line is not necessary since the "<protocol>" is explained immediately after and this placement makes it looks like a confusing bit of literal squid.conf text.
```suggestion
```

Reply:
commit a960091bbd
This commit also fixes the http_upgrade_request_protocols text documentation to fit to a 80 cols terminal